velox/docs/configs.rst

```diff
@@ -760,3 +760,74 @@ Tracing
      - integer
      - 0
      - The max trace bytes limit. Tracing is disabled if zero.
+
+Cache
+-----
+.. list-table::
+   :widths: 30 10 10 70
+   :header-rows: 1
+
+   * - Property Name
+     - Type
+     - Default Value
+     - Description
+   * - async-data-cache-enabled
+     - bool
+     - true
+     - If true, enable async data cache.
+   * - async-cache-ssd-gb
+     - integer
+     - 0
+     - The size of the SSD.
+   * - async-cache-ssd-path
+     - string
+     - /mnt/flash/async_cache.
+     - The directory that is mounted onto SSD.
+   * - async-cache-max-ssd-write-ratio
+     - double
+     - 0.7
+     - The max ratio of the number of in-memory cache entries being written to SSD cache over the total number of cache entries. This is to control SSD cache write rate, and once the ratio exceeds this threshold, then we stop writing to SSD cache.
+   * - async-cache-max-ssd-savable-ratio
+     - double
+     - 0.125
+     - The min ratio of SSD savable (in-memory) cache space over the total cache space. Once the ratio exceeds this limit, we start writing SSD savable cache entries into SSD cache.
+   * - async-cache-max-ssd-savable-bytes
+     - integer
+     - 1 << 24 (16 MB)
+     - Min SSD savable (in-memory) cache space to start writing SSD savable cache entries into SSD cache. NOTE: we only write to SSD cache when both above conditions are satisfied.
+   * - async-cache-persistence-interval
+     - string
+     - 0s
```

Comment:
Will add units has to be in seconds here.

Reply:
The unit is part of the string and we can see it is in seconds. But we would allow time based units like hour, day etc to be used as well. Not just seconds and we want to mention this in the description what they are.

Reply:
Added 
The following time units are supported: ns, us, ms, s, m, h, d.

Reference: https://github.com/facebookincubator/velox/blob/main/velox/common/config/Config.cpp#L82